netci.groovy

```diff
@@ -285,6 +285,41 @@ def osShortName = ['Windows 10': 'win10',
 }
 
 // **************************
+// Define perf testing.  These tests should be run daily, and will run on Jenkins not in Helix
+// **************************
+[true, false].each { isPR ->
+    ['Release'].each { configurationGroup ->
+        ['Windows_NT'].each { os ->
+            def osGroup = osGroupMap[os]
+            def newJobName = "perf_${os.toLowerCase()}_${configurationGroup.toLowerCase()}"
+
+            def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
+                // On Windows we use the packer to put together everything. On *nix we use tar
+                steps {
+                    batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd /p:ConfigurationGroup=${configurationGroup} /p:OSGroup=${osGroup} /p:Performance=true")
+                    batchFile("C:\\Packer\\Packer.exe .\\bin\\build.pack .\\bin")
+                }
+            }
+
+            // Set the affinity.
+            Utilities.setMachineAffinity(newJob, os, 'latest-or-auto')
```

Comment:
Running daily perf tests on **any** machine will give unreliable results, won't it? It seems like you'd want some fixed perf machines that have the same hardware and configuration and don't run any other tests. If our between-build cleanup is *really* thorough then this probably won't affect our results too much, but it's worth considering.